pkgs/by-name/fi/firebase-tools/package.nix

```diff
@@ -35,6 +36,8 @@ buildNpmPackage rec {
     PUPPETEER_SKIP_DOWNLOAD = true;
   };
 
+  passthru.updateScript = nix-update-script { };
```

Comment:
This is unnecessary, r-ryantm can handle updates without an explicit update script.

Reply:
Fixed.